EIPS/eip-tbd.md

```diff
@@ -0,0 +1,74 @@
+---
+eip: <to-be-assigned>
+title: Big integer modular exponentiation (EIP-198) gas cost
```

Comment:
I think the title would be nicer if it states the intent, such as "Cost adjustment/Repricing/Review of the Big integer modular exponentiation precompile (EIP-198)".

I would however suggest to go for something shorter like "Repricing of the EIP-198 modexp precompile".